<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompetitionViewer</class>
 <widget class="QWidget" name="CompetitionViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Организатор</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDateEdit" name="deBegin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="date">
        <date>
         <year>1998</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QDateEdit" name="deEnd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="date">
        <date>
         <year>2018</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Вид спорта</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Сооружение</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>по:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>c:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1" colspan="5">
      <widget class="QCheckBox" name="cbDateRange">
       <property name="text">
        <string>Диапазон дат</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="5">
      <widget class="QComboBox" name="cbSponsor"/>
     </item>
     <item row="1" column="1" colspan="5">
      <widget class="QComboBox" name="cbSport"/>
     </item>
     <item row="0" column="1" colspan="5">
      <widget class="QComboBox" name="cbFacility"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
